Failing test on Windows

`cytominer-database` can easily be installed with pip on Windows. See:  [installation instructions](https://github.com/cytomining/cytominer-database/wiki/Development#installation). 

However, the following error occurs after incorporating [this change](https://github.com/cytomining/cytominer-database/pull/43/files#diff-2f2baabd84a3efbea9343e91ce0ffa85):

```
C:\...\cytominer-database [dependencies +0 ~2 -0 !]> pytest
============================= test session starts =============================
platform win32 -- Python 2.7.14, pytest-3.2.2, py-1.4.34, pluggy-0.4.0 -- c:\python27\python.exe
cachedir: .cache
rootdir: C:\...\cytominer-database, inifile: setup.cfg
collected 2 items

tests/test_ingest.py::test_seed[htqc] FAILED
tests/test_munge.py::test_munge[htqc] PASSED

================================== FAILURES ===================================
_______________________________ test_seed[htqc] _______________________________

dataset = {'data_dir': 'tests/data_a', 'ingest': {'Cells_ncols': 294, 'Cells_nrows': 40, 'Cytoplasm_ncols': 279, 'Cytopl
asm_nrows': 40, ...}, 'munge': True, 'munged_dir': 'tests/data_a_munged', ...}

    def test_seed(dataset):
        config_file = os.path.join(dataset["data_dir"], "config.ini")

        config = configparser.ConfigParser()

        config.read(config_file)

        if dataset["munge"]:
            cytominer_database.munge.munge(config, dataset["data_dir"])

        with backports.tempfile.TemporaryDirectory() as temp_dir:
            sqlite_file = os.path.join(temp_dir, "test.db")

>           cytominer_database.ingest.seed(config=config, source=dataset["data_dir"], target="sqlite:///{}".format(str(s
qlite_file)))

tests\test_ingest.py:28:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
cytominer_database\ingest.py:115: in seed
    into(input=image, output=target, name=name.capitalize(), identifier=identifier)
cytominer_database\ingest.py:85: in into
    odo.odo(source, "{}::{}".format(output, name), has_header=True)
c:\python27\lib\site-packages\odo\odo.py:91: in odo
    return into(target, source, **kwargs)
c:\python27\lib\site-packages\multipledispatch\dispatcher.py:164: in __call__
    return func(*args, **kwargs)
c:\python27\lib\site-packages\odo\into.py:43: in wrapped
    return f(*args, **kwargs)
c:\python27\lib\site-packages\odo\into.py:149: in into_string_string
    return into(a, resource(b, **kwargs), **kwargs)
c:\python27\lib\site-packages\multipledispatch\dispatcher.py:164: in __call__
    return func(*args, **kwargs)
c:\python27\lib\site-packages\odo\into.py:43: in wrapped
    return f(*args, **kwargs)
c:\python27\lib\site-packages\odo\into.py:143: in into_string
    return into(a, b, dshape=dshape, **kwargs)
c:\python27\lib\site-packages\multipledispatch\dispatcher.py:164: in __call__
    return func(*args, **kwargs)
c:\python27\lib\site-packages\odo\into.py:43: in wrapped
    return f(*args, **kwargs)
c:\python27\lib\site-packages\odo\into.py:131: in into_object
    return append(target, source, dshape=dshape, **kwargs)
c:\python27\lib\site-packages\multipledispatch\dispatcher.py:171: in __call__
    return func(*args, **kwargs)
c:\python27\lib\site-packages\odo\backends\sql.py:493: in append_anything_to_sql_Table
    return append(t, convert(Iterator, o, **kwargs), **kwargs)
c:\python27\lib\site-packages\odo\core.py:44: in __call__
    return _transform(self.graph, *args, **kwargs)
c:\python27\lib\site-packages\odo\core.py:57: in _transform
    ooc_types=ooc_types)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

graph = <networkx.classes.graphviews.SubDiGraph object at 0x057D5230>
source = <class 'odo.backends.csv.CSV'>, target = <class '_abcoll.Iterator'>
excluded_edges = set([])
ooc_types = set([<class '_abcoll.Iterator'>, <class 'pandas.io.pytables.AppendableFrameTable'>, <class 'odo.chunks.Chunk
s'>, <class 'sqlalchemy.sql.schema.Table'>, <class 'odo.backends.csv.CSV'>, <class 'odo.backends.json.JSONLines'>])

    def path(graph, source, target, excluded_edges=None, ooc_types=ooc_types):
        """ Path of functions between two types """
        if not isinstance(source, type):
            source = type(source)
        if not isinstance(target, type):
            target = type(target)

        if source not in graph:
            for cls in valid_subclasses:
                if issubclass(source, cls):
                    source = cls
                    break

        # If both source and target are Out-Of-Core types then restrict ourselves
        # to the graph of out-of-core types
        if ooc_types:
            oocs = tuple(ooc_types)
            if issubclass(source, oocs) and issubclass(target, oocs):
                graph = graph.subgraph([n for n in graph.nodes()
                                        if issubclass(n, oocs)])
        with without_edges(graph, excluded_edges) as g:
            pth = nx.shortest_path(g, source=source, target=target, weight='cost')
            result = [(src, tgt, graph.edge[src][tgt]['func'])
>                     for src, tgt in zip(pth, pth[1:])]
E           AttributeError: 'SubDiGraph' object has no attribute 'edge'

c:\python27\lib\site-packages\odo\core.py:94: AttributeError
---------------------------- Captured stdout call -----------------------------
File tests\data_a\dummy_dir\object.csv does not exist
===================== 1 failed, 1 passed in 3.59 seconds ======================
```

--- a/cytominer_database/utils.py
+++ b/cytominer_database/utils.py
@@ -44,7 +44,7 @@ def validate_csv(csvfile):
     cmd = "csvclean -n {}".format(csvfile)
 
     try:
-        csvcheck = subprocess.check_output(cmd, shell=True)
+        csvcheck = subprocess.check_output(cmd, shell=True).strip()
 
     except subprocess.CalledProcessError as e:
         if e.returncode == 1:
@@ -60,7 +60,7 @@ def validate_csv(csvfile):
 
     file_size = os.stat(csvfile).st_size
 
-    return (file_size > 0) & (nrows >= 1) & (csvcheck == b'No errors.\n')
+    return (file_size > 0) & (nrows >= 1) & (csvcheck == b'No errors.')
 
 
 def validate_csv_set(config, directory):

--- a/setup.py
+++ b/setup.py
@@ -46,6 +46,7 @@ setuptools.setup(
         'click>=6.7',
         'configparser>=3.5.0',
         'csvkit>=1.0.2',
+        'networkx<2.0',  # TODO: https://github.com/blaze/odo/issues/579 -- remove dependency when resolved
         'odo>=0.5.0',
         'pandas>=0.20.3'
     ],

